### PR TITLE
Odv

### PR DESCRIPTION
**Pull request by Marie59**
Hi @bgruening ! I just had the approval of ODV's author to put ODV interactive tool on galaxy. So, I initiate this PR to add the tool to the successful ones from the Lille training. Tell me if there is anything to change.

### CODE REVIEW

File: tools/interactive/interactivetool_odv.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<tool id="interactive_tool_odv" tool_type="interactive" name="ODV" version="@VERSION@">
+    <description>interactive plotting tool for geo-referenced data</description>
+    <macros>
+    <token name="@VERSION@">5.6.5</token>
+    </macros>
+    <requirements>
+        <container type="docker">quay.io/mariejo/docker_odv</container>
```
Comment (bgruening) [inline tools/interactive/interactivetool_odv.xml]: Please note that the containers should be versioned ideally.

File: tools/interactive/interactivetool_odv.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <collection name="outputs" type="list" label="ODV outputs">
+            <discover_datasets pattern="__name_and_ext__" directory="output" />
+        </collection>
+    </outputs>
+    <tests>
+    </tests>
+    <help><![CDATA[
+        `ODV <https://odv.awi.de/>`_ plots geo-referenced and other arrays from netCDF, HDF, GRIB, and other datasets.
+    ]]>
+    </help>
```
Comment (bgruening) [inline tools/interactive/interactivetool_odv.xml:37]: Do you want to add maybe a citation for the tool?

File: tools/interactive/interactivetool_odv.xml (hunk 1 above)
Comment (Marie59) [reply tools/interactive/interactivetool_odv.xml]: Sorry it's my 1st time that I push a container online. Should I fix that ?

File: tools/interactive/interactivetool_odv.xml (hunk 2 above)
Comment (Marie59) [reply tools/interactive/interactivetool_odv.xml:37]: Ah maybe yes I will search for the right one. Thanks !!

File: tools/interactive/interactivetool_odv.xml (hunk 1 above)
Comment (bgruening) [reply tools/interactive/interactivetool_odv.xml]: Would be nice you can do `docker tag quay.io/mariejo/docker_odv quay.io/mariejo/docker_odv:5.6.5` and than `docker push quay.io/mariejo/docker_odv quay.io/mariejo/docker_odv:5.6.5`

File: tools/interactive/interactivetool_odv.xml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<tool id="interactive_tool_odv" tool_type="interactive" name="ODV" version="@VERSION@">
+    <description>interactive plotting tool for geo-referenced data</description>
+    <macros>
+    <token name="@VERSION@">5.6.5</token>
+    </macros>
+    <requirements>
+        <container type="docker">quay.io/mariejo/docker_odv:5.6.5</container>
```
Comment (bgruening) [inline tools/interactive/interactivetool_odv.xml]:
```suggestion
        <container type="docker">quay.io/mariejo/docker_odv:@VERSION@</container>
```

**Comment by Marie59** [comment]
Thanks a lot @bgruening !!

**Comment by bgruening** [comment]
Unless our @usegalaxy-eu/admin are not deploying it earlier, this tool will be available tomorrow.

Enjoy!

**Pull request merged**